src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RoundDownT_ConsistentOneColumnTableResultDisabled.txt

```diff
@@ -0,0 +1,37 @@
+// RoundDown(Number_or_table, digits_or_table), towards zero
+
+// Table and scalar
+
+>> RoundDown([1.2, 3.4, 5.6, 7.8], 0)
+Table({Value:1},{Value:3},{Value:5},{Value:7})
+
+>> RoundDown([1.23, 4.56, 7.89, 10.12], 1)
+Table({Value:1.2},{Value:4.5},{Value:7.8},{Value:10.1})
+
+>> RoundDown([-9.8765, -8.7654, -7.6543, -6.5432, -5.4321], 2)
+Table({Value:-9.87},{Value:-8.76},{Value:-7.65},{Value:-6.54},{Value:-5.43})
+
+>> RoundDown(Filter([1,2,3], Value > 10), 0) // Empty table
+Table()
+
+// Scalar and table
+>> RoundDown(1234.5678, [-2, -1, 0, 1, 2])
+Table({Result:1200},{Result:1230},{Result:1234},{Result:1234.5},{Result:1234.56})
```

Comment:
Given that we already have shipped ConsistentOneColumnTable support in PA now (It's always ON), do we still need to add tests for both enable and disable support?